### PR TITLE
Show Claims and Appeals errors

### PR DESCRIPTION
**Pull request by erikphansen**
## Description
- Show error when there is a problem fetching either claims or appeals data
- Show error when there is known downtime affecting claims or apeals

## Testing done
Unit and cypress tests

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/personalization/dashboard-2/tests/e2e/downtime.cypress.spec.js [hunk 1]
```diff
@@ -0,0 +1,191 @@
+import disableFTUXModals from '~/platform/user/tests/disableFTUXModals';
```
Comment (erikphansen) [inline src/applications/personalization/dashboard-2/tests/e2e/downtime.cypress.spec.js:1]: This file was renamed from `upcoming-downtime`...

File: src/applications/personalization/dashboard-2/tests/e2e/downtime.cypress.spec.js [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+            description: '',
+          },
+        },
+      ],
+    });
+    cy.visit(manifest.rootUrl);
+    cy.findByRole('heading', { name: /My VA/i }).should('exist');
+    cy.findByRole('button', { name: /continue/i }).should('not.exist');
+  });
+  it('should show an alert in place of Claims and Appeals data if there is active MHV service downtime', () => {
```
Comment (erikphansen) [inline src/applications/personalization/dashboard-2/tests/e2e/downtime.cypress.spec.js:86]: This is where the actual new code starts.

File: src/applications/personalization/dashboard-2/components/Dashboard.jsx [hunk 3]
```diff
@@ -121,20 +147,30 @@ const Dashboard = ({
 
               {showHealthCareError ? (
                 <div className="vads-l-row">
-                  <div className="vads-l-col--12 medium-screen:vads-l-col--8">
+                  <div className="vads-l-col--12 medium-screen:vads-l-col--8 medium-screen:vads-u-padding-right--3">
```
Comment (sanlouise) [inline src/applications/personalization/dashboard-2/components/Dashboard.jsx:150]: Just a personal preference -- I would do 

```
showHealthCareError && (

) 
```

So you don't need the `null` - but it's fine either way 🙂

File: src/applications/personalization/dashboard-2/components/Dashboard.jsx [hunk 4]
```diff
@@ -145,12 +181,21 @@ const Dashboard = ({
   );
 };
 
+const isClaimsAvailableSelector = createIsServiceAvailableSelector(
```
Comment (sanlouise) [inline src/applications/personalization/dashboard-2/components/Dashboard.jsx:184]: Is this the best place for these selectors to live?

**Review by sanlouise — APPROVED** [review]
Looks good to me! 💯 👍

File: src/applications/personalization/dashboard-2/components/Dashboard.jsx (hunk 3 above)
Comment (erikphansen) [reply src/applications/personalization/dashboard-2/components/Dashboard.jsx:150]: I know, I think we talked about this sometime in the past month 😆  [It's just a little safer to use ternaries in JSX](https://kentcdodds.com/blog/use-ternaries-rather-than-and-and-in-jsx) so I'm trying to get into that habit.

File: src/applications/personalization/dashboard-2/components/Dashboard.jsx (hunk 4 above)
Comment (erikphansen) [reply src/applications/personalization/dashboard-2/components/Dashboard.jsx:184]: Perhaps not, I _just_ added this quickly, copying [how we are already doing it in the ClaimsAndAppeals component](https://github.com/department-of-veterans-affairs/vets-website/blob/8be90e8bad321a67ce4e593111cb0f0d4e4b822a/src/applications/personalization/dashboard-2/components/claims-and-appeals/ClaimsAndAppeals.jsx), which also might not be doing it in the best place? Would you want to move it to live _in_ `mapStateToProps`? Or someplace else?

**Pull request merged**